chart/env/prod.yaml

```
@@ -61,7 +61,8 @@ common:
   hfEndpoint: "https://huggingface.co"
   # Log level
   logLevel: "INFO"
-
+  contentMaxSize: 1000000
```

```suggestion
  contentMaxSize: 1_000_000
```

Is it the value we want in production, btw?

---

I fixed it to 10_000_000